src/NuGet.Clients/PackageManagement.UI/Feeds/MultiSourcePackageFeed.cs

```diff
@@ -151,12 +151,18 @@ public MultiSourcePackageFeed(IEnumerable<SourceRepository> sourceRepositories,
 
             if (notCompleted.Any())
             {
-                var items = notCompleted
-                        .ToDictionary(kv => kv.Key, kv => GetLoadingStatus(kv.Value.Status));
+                var items = notCompleted.ToDictionary(
+                    kv => kv.Key,
+                    kv => new { LoadingStatus = GetLoadingStatus(kv.Value.Status), kv.Value.Exception });
```

Comment:
Style issue or what? I will make the change but both solutions seems fine to me.